tdrs-backend/tdpservice/conftest.py

```diff
@@ -269,10 +269,10 @@ def base_regional_data_file_data(fake_file_name, regional_user):
         "extension": "txt",
         "section": "Active Case Data",
         "user": str(regional_user.id),
-        "region": regional_user.region.id,
+        "regions": [regional_user.regions.first().pk],
```

Comment:
nit: suggest to make query specific, .first() might pass now, but might not pass later